### ISSUE TITLE
Tooltip not working :(

### ISSUE THREAD

**Issue by jgphilpott** [jacob-j/vscode-folder-path-color#7]
The tooltip feature is not working.

**Comment by jacob-j** [comment]
Thank you @jgphilpott for the fix, I have merged and deploying a new version now (0.0.13) :)

**Comment by jgphilpott** [comment]
@jacob-j no problem! If you have the time let me know what you think of the other issue I opened: https://github.com/jacob-j/vscode-folder-path-color/issues/6

Basically, I want the git decoration colors to continue working on folders in addition to files. I also raised [this issue with the VS Code team](https://github.com/microsoft/vscode/issues/187756) cuz it might be a problem with the API, not your extension. If a workaround is possible though I would love to figure it out!
